app/models/site.rb

```diff
@@ -36,6 +36,7 @@ class Site < ApplicationRecord
   validates :name, :slug, :domain, presence: true
   validates :place_name unless :default_site?
   validates :hero_text, length: { maximum: 120 }
+  validates :domain, format: { with: %r{\Ahttps://[^\s,]+\z}, message: 'A domain must start with "https://"' }
```

Comment:
We do also support `webcal://`.

I would probably just validate looking for `https://` and then anything after that is probably okay. Also there was some drama about supporting plain `http`.